index 81758cf..ed2a02a 100644
--- a/kernel/private/classes/ezpkernelweb.php
+++ b/kernel/private/classes/ezpkernelweb.php
@@ -651,6 +651,7 @@ class ezpKernelWeb implements ezpKernelHandler
 
             if ( !$displayMissingModule && $moduleCheck['result'] && $this->module instanceof eZModule )
             {
+                ezpEvent::getInstance()->notify( 'module/start', array( $moduleName, $functionName, $params ) );
                 // Run the module/function
                 eZDebug::addTimingPoint( "Module start '" . $this->module->attribute( 'name' ) . "'" );