index 73197ee..dfbcba7 100644
--- a/index.php"
+++ b/index.php"
@@ -581,6 +580,7 @@ while ( $moduleRunRequired )
          $moduleCheck['result'] &&
          $module instanceof eZModule )
     {
+        ezpEvent::getInstance()->notify( 'module/start', array( $module, $function_name, $params ) );
         // Run the module/function
         eZDebug::addTimingPoint( "Module start '" . $module->attribute( 'name' ) . "'" );
 