diff --git "a/C:\\Users\\kristina\\AppData\\Local\\Temp\\TortoiseGit\\indFE15.tmp\\index-e73cf66-left.php" "b/C:\\workspace\\madsack_dev\\extension\\xrowakamai\\index.php"
index 73197ee..dfbcba7 100644
--- "a/C:\\Users\\kristina\\AppData\\Local\\Temp\\TortoiseGit\\indFE15.tmp\\index-e73cf66-left.php"
+++ "b/C:\\workspace\\madsack_dev\\extension\\xrowakamai\\index.php"
@@ -581,6 +580,7 @@ while ( $moduleRunRequired )
          $moduleCheck['result'] &&
          $module instanceof eZModule )
     {
+        ezpEvent::getInstance()->notify( 'module/start', array( $module, $function_name, $params ) );
         // Run the module/function
         eZDebug::addTimingPoint( "Module start '" . $module->attribute( 'name' ) . "'" );
 